### ISSUE TITLE
Keypad does not accept input from Wacom Intuos (12x18) + 'fix'

### ISSUE THREAD

**Issue by rsnel** [CE-Programming/CEmu#348]
**Operating System:** Debian GNU/Linux 10 
**CEmu version:** v2.0dev (cee9c86c7e899c8ba99360224220c658e1db22f1)
**Describe your issue:**  
  Keypad does not accept input from Wacom Intuos (12x18). I expect to be able to use the table as I can use the mouse.

What are the steps to reproduce this issue?
-------------------------------------------
1. Attempt to press keys using the graphics tablet

Any logs, error output, screenshot, other comments...?
------------------------------------------------------

If I remove the first 4 lines of the function

    void KeypadWidget::mouseEvent(QMouseEvent *event) {

then the touchpad works as expected. However, I still wanted to report this, since those lines must have been there for a reason.

**Comment by adriweb** [comment]
I don't believe any of us has a graphics tablet to help debug this, it would be nice to know what the `source()` is for this.
Apparently it can be those values: https://doc.qt.io/qt-5/qt.html#MouseEventSource-enum

@jacobly0 added a check for a specific one so there must be a reason...?

**Comment by CollinScripter** [comment]
This occurs on other graphics tablets as well.
I have a Huion Inspiroy H640P which is ~40 USD and it doesn't work correctly either.

**Comment by rsnel** [comment]
I have recompiled the code with a printf() statement to determine the value of source() when using the tablet. The value is 

Qt::MouseEventSynthesizedByQt  2

**Comment by adriweb** [comment]
Right, this corresponds to "synthesized from an unhandled touch event", hmmm...

**Comment by adriweb** [comment]
If someone is able to try the [latest nightly release](https://github.com/CE-Programming/CEmu/releases/tag/nightly) (since Qt6 may have already changed the behavior here), it would be nice.
